internal/forge/gitea/connector.go

```diff
@@ -206,7 +206,7 @@ func (self Connector) squashMergeProposal(number int, message gitdomain.CommitMe
 	return err
 }
 
-func (self Connector) updateProposalBodyFn(proposalData forgedomain.ProposalInterface, updatedBody string) error {
+func (self Connector) updateProposalBody(proposalData forgedomain.ProposalInterface, updatedBody string) error {
```

Comment:
Thanks for fixing this! :pray: This change is orthogonal to the other changes in this pull request, so I'm going to extract it into its own pull request.